src/js/pensions/config/form.js

```diff
@@ -133,17 +138,62 @@ const formConfig = {
       }
 
     },
+    workHistory: {
+      title: 'Work history',
+      pages: {
+        disabilityHistory: {
+          title: 'Disability history',
+          path: 'disability/history',
+          depends: (formData) =>
+            moment().startOf('day').subtract(65, 'years').isBefore(formData.veteranDateOfBirth),
+          uiSchema: {
+            disabilities: {
+              'ui:title': 'What Disabilities prevent you from working?',
+              'ui:order': ['name', 'disabilityStartDate'],
+              'ui:options': {
+                viewField: DisabilityField
+              },
+              items: {
+                name: {
+                  'ui:title': 'Disability'
+                },
+                disabilityStartDate: dateUI('Date disability began')
+              }
+            },
+            // TODO: update schema with this field if stakeholders approve
+            hasVisitedVAMC: {
+              'ui:title': 'Have you been treated at a VA Medical Center for the above disability?',
+              'ui:widget': 'yesNo'
+            }
+          },
+          schema: {
+            type: 'object',
+            required: ['disabilities', 'hasVisitedVAMC'],
+            properties: {
+              disabilities: {
+                type: 'array',
+                minItems: 1,
+                items: {
+                  type: 'object',
+                  required: ['name', 'disabilityStartDate'],
+                  properties: disabilities.items.properties
+                }
+              },
+              hasVisitedVAMC: {
+                type: 'boolean'
+              }
+            }
+          }
+        }
+      }
+    },
     financialDisclosure: {
       title: 'Financial Disclosure',
       pages: {
         netWorth: {
           path: 'financial-disclosure/net-worth',
           title: item => `${item.veteranFullName.first} ${item.veteranFullName.last} net worth`,
           initialData: {
-            veteranFullName: {
```

Comment:
Why remove the veteranFullName initialData but not the spouseFullName initialData? Is this based on tests written so far?